src/go/k8s/controllers/redpanda/console_controller_test.go

```diff
@@ -53,23 +54,26 @@ var _ = Describe("Console controller", func() {
 
 		timeout  = time.Second * 30
 		interval = time.Millisecond * 100
+
+		deploymentImage      = "vectorized/console:latest"
+		enableSchemaRegistry = true
+		enableConnect        = false
 	)
 
-	Context("When creating Console", func() {
+	BeforeEach(func() {
```

Comment:
Why there is no `AfterEach` where cluster and console is clean up?

Reply:
This envtest just runs on process and will end after the test. I don't think we need to clean up.

Reply:
Then the whole test suite is a giant single test. One test depends on other to not malform the envtest state.